app/views/posts/show.html.erb

```diff
@@ -1,7 +1,17 @@
-<h1>This page shows the content of a blog post</h1>
-<ul>It shows
-<li>Blog post</li>
-<li>Number of comments</li>
-<li>Number of likes</li>
-<li>All comments</li>
-</ul>
\ No newline at end of file
+<section class="user-post">
+    <div class="post-content">
+        <% user = User.find(@user_post.author_id) %>
+        <p class="post-stats"><span>Post <%= @user_post.id %> by <strong><%= user.name %></strong> </span><span>Comments: <%= @user_post.comments_counter %>, Likes: <%= @user_post.likes_counter %></span></p>
```

Comment:
- [ ] You already have the `create` action in the likes controller and setup the route for liking a post, however, a user still needs to be able to like a post by clicking either a link or a button as you can see in the requirement below:
![image](https://github.com/danielochuba/duo-blog/assets/46839548/7233ef25-702d-4112-8df5-aa384871b897)


### HINT 🔔 
You can create a button that triggers the like action like this 👇 
```erb
<%= button_to "Like", post_likes_path(user_id: current_user, post_id: @post.id), method: :post, remote: true %>
```